test/hexpm/repository/package_test.exs

```diff
@@ -223,4 +223,21 @@ defmodule Hexpm.Repository.PackageTest do
              |> Repo.all()
              |> Enum.map(& &1.id)
   end
+
+  test "sort packages by recent releases", %{organization: organization} do
+    %{id: ecto_id} = insert(:package, organization_id: organization.id)
+    %{id: phoenix_id} = insert(:package, organization_id: organization.id)
+    %{id: decimal_id} = insert(:package, organization_id: organization.id)
+
+    insert(:release, package_id: phoenix_id)
+    insert(:release, package_id: decimal_id)
+    insert(:release, package_id: ecto_id)
```

Comment:
Please insert an additional release for a package to ensure we get distinct packages in the results.

Reply:
@ericmj this was a really good suggestion - the query did have a problem (didn't return distinct packages) 👍 It's working as expected now.

P.S. Thanks again @michalmuskala for saving me hours of debugging on the query 😊